src/features/Students/StudentsFilters/index.jsx

```diff
@@ -0,0 +1,69 @@
+import { Col, Form } from '@edx/paragon';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Filters = props => {
+  const {
+    filters,
+    setFilters,
+  } = props;
+
+  const handleInputChange = (e) => {
+    setFilters({
+      ...filters,
+      [e.target.name]: e.target.value.trim(),
+    });
+  };
+
+  return (
+    <Form className="row justify-content-center">
+      <Form.Group as={Col}>
+        <Form.Control
+          type="text"
+          floatingLabel="Name"
+          name="learner_name"
+          placeholder="Enter Student Name"
+          value={filters.learner_name}
+          onChange={handleInputChange}
+          className="mb-3"
+        />
+        <Form.Control
+          type="email"
+          floatingLabel="Email"
+          name="learner_email"
+          placeholder="Enter Student Email"
+          value={filters.learner_email}
+          onChange={handleInputChange}
+          className="mb-3"
+        />
+        <Form.Control
+          type="text"
+          floatingLabel="Instructor"
+          name="instructor"
+          placeholder="Enter Instructor"
+          value={filters.instructor}
+          onChange={handleInputChange}
+          className="mb-3"
+        />
+        <Form.Control
+          type="text"
+          floatingLabel="Class Name"
+          name="ccx_name"
+          placeholder="Enter Class Name"
```

Comment:
You are right, I think the Class Name is clearer, because in the platform it is known as course name or ccx name. So I changed the name in the column, do you agree?

Reply:
@anfbermudezme yes, It's ok. Please, update screenshot to see that changes.

Reply:
Done

Reply:
@anfbermudezme you're missing update the first screenshot.